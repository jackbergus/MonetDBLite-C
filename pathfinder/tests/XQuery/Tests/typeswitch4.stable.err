stderr of test 'typeswitch4` in directory 'tests/XQuery` itself:


# 15:53:14 >  
# 15:53:14 >  Mtimeout -timeout 180 Mserver "--config=/local_tmp/MonetDB/Linux/etc/MonetDB.conf" --debug=10 --set "monet_mod_path=/local_tmp/pathfinder/Linux/lib/MonetDB:/local_tmp/MonetDB/Linux/lib/MonetDB" --set "gdk_dbfarm=/local_tmp/MonetDB/Linux/var/MonetDB/dbfarm" --set "sql_logdir=/local_tmp/MonetDB/Linux/var/MonetDB/log" --set mapi_port=33498 --set monet_prompt= --trace "--dbname=mTests_tests_XQuery" --dbinit="module(pathfinder); pfstart();" ; echo ; echo Over..
# 15:53:14 >  


# 15:53:15 >  
# 15:53:15 >  Mtimeout -timeout 60 MapiClient -lxquery -sxml -umonetdb -Pmonetdb --host=localhost --port=33498 typeswitch4.xq
# 15:53:15 >  





# 15:53:16 >  
# 15:53:16 >  Done.
# 15:53:16 >  

