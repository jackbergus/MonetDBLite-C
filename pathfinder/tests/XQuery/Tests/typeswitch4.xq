for $a in (1, "foo", 2.3, 3E1, <a/>, attribute b {"b"}, text {"text"})
return typeswitch ($a)
        case xs:integer return "integer"
        case xs:double return "double"
        case xs:decimal return "decimal"
        case xs:string return "string"
        case attribute() return "attribute()"
        case element() return "element()"
        case text() return "text()"
        case node() return "node()"
        default return "null"

