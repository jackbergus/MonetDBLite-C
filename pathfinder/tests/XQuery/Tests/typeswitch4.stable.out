stdout of test 'typeswitch4` in directory 'tests/XQuery` itself:


# 15:53:14 >  
# 15:53:14 >  Mtimeout -timeout 180 Mserver "--config=/local_tmp/MonetDB/Linux/etc/MonetDB.conf" --debug=10 --set "monet_mod_path=/local_tmp/pathfinder/Linux/lib/MonetDB:/local_tmp/MonetDB/Linux/lib/MonetDB" --set "gdk_dbfarm=/local_tmp/MonetDB/Linux/var/MonetDB/dbfarm" --set "sql_logdir=/local_tmp/MonetDB/Linux/var/MonetDB/log" --set mapi_port=33498 --set monet_prompt= --trace "--dbname=mTests_tests_XQuery" --dbinit="module(pathfinder); pfstart();" ; echo ; echo Over..
# 15:53:14 >  

# Monet Database Server V4.11.3
# Copyright (c) 1993-2006, CWI. All rights reserved.
# Compiled for i686-suse-linux/32bit with 32bit OIDs; dynamically linked.
# Visit http://monetdb.cwi.nl/ for further information.

printf("\nReady.\n");

Ready.
quit();

Over..

# 15:53:15 >  
# 15:53:15 >  Mtimeout -timeout 60 MapiClient -lxquery -sxml -umonetdb -Pmonetdb --host=localhost --port=33498 typeswitch4.xq
# 15:53:15 >  

<?xml version="1.0" encoding="utf-8"?>
<XQueryResult>integer string decimal double element() attribute() text()</XQueryResult>

# 15:53:16 >  
# 15:53:16 >  Done.
# 15:53:16 >  

